pipelines/tekton/aiedge-e2e/templates/credentials-image-registry.secret.yaml.template

```diff
@@ -11,5 +11,6 @@ metadata:
     tekton.dev/docker-0: https://quay.io
 type: kubernetes.io/basic-auth
 stringData:
-  username: {{ IMAGE_REGISTRY_USERNAME }}
-  password: {{ IMAGE_REGISTRY_PASSWORD }}
+  username: "{{ IMAGE_REGISTRY_USERNAME }}"
+  password: "{{ IMAGE_REGISTRY_PASSWORD }}"
```

Comment:
Added quotes to some of the template files as it was breaking the go json parser. I think this shouldn't break the shell tests but I have not tested it